src/controllers/authentication.js

```diff
@@ -1,55 +1,69 @@
 import jwt from 'jsonwebtoken';
 import uuid from 'uuid';
 
-export function authenticationController(credentials, secretCode) {
-    return (req, res) => {
+export function authenticationController(secretCode) {
+    return async (req, res) => {
         const userInfo = req.body;
+        const userResponse = await req.context.models.User.findAll({
+            where: {
+                name: userInfo.name,
+                password: userInfo.password
```

Comment:
Do you store passwords not encrypted?
https://stackoverflow.com/questions/1054022/best-way-to-store-password-in-database